openspp-project/openspp-registry
# Spp dashboard base

## **Why is this change needed?**

To have a base dashboard components

## **How was the change implemented?**

Added new module named spp_dashboard_base where dashboard components are defined.

## **New unit tests**

None

## **Unit tests executed by the author**

None

## **How to test manually**

- Go to Apps Menu and search spp_dashboard_base
- Install module
- Check if installed successfully.

This is only a base module, functionalities can only be tested when another module is using it

## **Related links**
